test/dynamo/test_autograd_function.py

```diff
@@ -214,6 +214,24 @@ def forward(self, x):
         return self.f(x)
 
 
+class CustomTupleResultAutogradFunc(torch.autograd.Function):
+    # Test tuple return
+    @staticmethod
+    def forward(ctx, foo):
+        result = foo + foo
+        result = result + foo
+        return (result, result, result)
+
+    @staticmethod
+    def backward(ctx, grad_output):
+        return grad_output
```

Comment:
backward should accept a number of grad_output equal to the number of outputs (3)

Reply:
got it